arrow-site/doc/docs/arrow-core/datatypes/optiont/README.md

```diff
@@ -12,17 +12,17 @@ video: EWfxL9yBUJo
 {:.advanced}
 advanced
 
-`OptionT` also known as the `Option` monad transformer allows to compute inside the context when `Option` is nested in a different monad.
+`OptionT`, also known as the `Option` monad transformer, allows computation inside the context when `Option` is nested in a different monad.
 
 One issue we face with monads is that they don't compose. This can cause your code to get really hairy when trying to combine structures like `ObservableK` and `Option`. But there's a simple solution, and we're going to explain how you can use Monad Transformers to alleviate this problem.
```

Comment:
We shouldn't start a sentence with "But."

Instead we should consider using a comma, and then but.

Also, why is Monad Transformers capitalized?